lib/std/start.zig

```diff
@@ -469,7 +469,7 @@ fn posixCallMainAndExit(argc_argv_ptr: [*]usize) callconv(.C) noreturn {
         // Apply the initial relocations as early as possible in the startup
         // process.
         if (builtin.position_independent_executable) {
-            std.os.linux.pie.relocate(phdrs);
+            @call(.always_inline, std.os.linux.pie.relocate, .{phdrs});
```

Comment:
For the exact same reason that we can't access global variables like `std.os.linux.elf_aux_maybe` yet.